e2e_test/source_inline/commands.toml

```diff
@@ -0,0 +1,108 @@
+# This file contains commands used by the tests.
+
+[tasks.source-test-hook]
+private = true
+dependencies = ["check-risedev-env-file"]
+env_files = ["${PREFIX_CONFIG}/risedev-env"]
+
+# Note about the Kafka CLI tooling:
+# - Built-in Kafka console tools:
+#       Java based.
+#       Style example: kafka-topics.sh --bootstrap-server localhost:9092 --topic t --create
+#       Some limitations: cannot disable logging easily, cannot consume to end and then exit.
+# - kcat:
+#       C based (rdkafka)
+#       Some limitations: cannot do admin operations, only consume/produce.
+# - rpk:
+#       Golang based.
+#       Style example: RPK_BROKERS=localhost:9092 rpk topic create t
+[tasks.kafka-hook]
+private = true
+description = "Check if Kafka is started by RiseDev"
+dependencies = ["source-test-hook"]
+script = '''
+#!/usr/bin/env sh
+set -e
+
+if [ ! -d "${PREFIX_BIN}/kafka" ]; then
+    echo "Kafka is not installed in ${PREFIX_BIN}/kafka. Did you enable Kafka using $(tput setaf 4)\`./risedev configure\`$(tput sgr0)?"
+    exit 1
+fi
```

Comment:
Will this be problematic in CI (where we start Kafka in user-managed mode)?

Reply:
CI also downloaded Kafka to use kafka built in tools like `kafka/bin/kafka-topics.sh` 🤡.